src/library_idbfs.js

```diff
@@ -128,8 +131,20 @@ mergeInto(LibraryManager.library, {
             if (!cursor) {
               return callback(null, { type: 'remote', db: db, entries: entries });
             }
-
-            entries[cursor.primaryKey] = { timestamp: cursor.key };
+			
```

Comment:
please use 2 spaces for indentation, and not tabs or a different number of spaces.